elfpy/bots/bot_info.py

```diff
@@ -16,31 +15,23 @@ class AgentConfig:
 
     Attributes
     ----------
-    name : str
-        The name of the agent
     policy : str
         The agent's policy; should match the class name
-    number_of_bots : int
-        The number of bots of this type to spin up
-    trade_chance : float
-        Percent chance that a agent gets to trade on a given block
+    name : str
+        The name of the agent
     budget : Budget
         The budget for the agent
-    scratch : dict
-        Any parameters for custom bots should go here
+    number_of_agents : int
+        The number of bots of this type to spin up
+    init_kwargs : dict
+        A dictionary of keyword arguments for the policy constructor
     """
 
-    # TODO: We should be able to enable this again once we remove evm_bots
+    # lots of configs!
     # pylint: disable=too-many-instance-attributes
 
     policy: Type[BasePolicy]
-    index: int | None = None  # TODO: Make this required when we remove evm_bots
```

Comment:
???

Reply:
the TODO was dumb -- we don't use it anymore